### DISCUSSION TITLE
More realistic pointcloud noise model

### DISCUSSION THREAD

**Discussion by richardrl** [bulletphysics/bullet3#3923] [Q&A]
The pointclouds from the built-in pyBullet camera are quite clean. How do I get more realistic noise, such that these pointclouds look closer to real pointcloud cameras?

**Comment by atirehgram** [comment]
Hi, how do you get the point clouds?

**Reply by rohit-kumar-j** [reply]
@atirehgram See #4049

**Comment by rohit-kumar-j** [comment]
@richardrl you could add noise once the data is captured. This could simulate the noise of the real point cloud capture. 
Something like a + or -  5% perhaps...
```python3
new_reading = []
for i in range(len(reading)):
    read_w_err = reading[i] + (0.05*reading[i])
    new_reading.append(read_w_err)
```
